README.md

```diff
@@ -125,21 +124,95 @@ Once the prerequisites are available, VA Explorer can be installed and demonstra
 
     `pytest`
 
-#### Running the application
+### Running the application
 
 * Run the application server
 
     `python manage.py runserver 0.0.0.0:8000`
 
-    The server will be running at http://0.0.0.0:8000/
+The server will be running at http://0.0.0.0:8000/
 
-### Version History
+## Building/Running in Docker
+
+Django can run locally inside Docker. This will also set up postgres and redis and automatically configure `DATABASE_URL` and `CELERY_BROKER_URL` to use the docker images of postgres and redis.
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.local.yml up django postgres
+```
+
+The server will be running at http://0.0.0.0:5000/
+
```

Comment:
I was able to access the application without issue via these new instructions, however I didn't have a user to login with after. Should we consider adding auto super-user setup or perhaps instructions for setting that up instead?

Reply:
Yeah, I'll add instructions for creating a super user in docker.